freemocap/gui/qt/widgets/directory_view_widget.py

```diff
@@ -45,17 +46,17 @@ def __init__(self, top_level_folder_path: Union[str, Path], get_active_recording
         self._tree_view_widget.setAlternatingRowColors(True)
         self._tree_view_widget.setColumnWidth(0, 250)
 
-        if self._top_level_folder_path is not None:
-            self.set_folder_as_root(self._top_level_folder_path)
+        self._path_label = QLabel(str(self._gui_state.freemocap_data_folder_path))
```

Comment:
Is this where you were having issues with Window's paths losing their backspaces? 

It may be because you're casting it to a `str` (where \'s aren't rendered bc they are read as 'escape characters' for \n \t etc) 

Can we cast it straight to a `pathlib.Path()` instead of `str`? Does that work as expected? 

One of `pathlib`'s main jobs is to handle windows path backslashes lol 😂

Reply:
This is just for the QLabel, which has to have a str as input. We don't pull data out of the QLabel ever (we don't do `np.load(self._path_label.text)` for example), so even if it's not represented without backspaces, it won't affect anything

Reply:
Ah, right, my bad. Makes sense